.github/workflows/cbmc-latest.yml

```diff
@@ -93,4 +93,4 @@ jobs:
       - name: Execute Kani performance ignored tests
         working-directory: ./kani
         continue-on-error: true
-        run: cargo run -p compiletest -- --suite perf --mode cargo-kani-test ignore --ignored
+        run: cargo run -p compiletest -- --suite perf --mode cargo-kani-test ignore --ignored --no-fail-fast
```

Comment:
Do we need both the `ignore` and `--ignored` arguments?

Reply:
Yes, we do. The "ignore" filters all tests that don't have ignore in their name. The `--ignored` tells rust to execute tests even if they are marked as an ignore test.

I.e., without the `ignore` argument, all tests will run. Without the `--ignored`, the ignore tests will be ignored. 🙂